### ISSUE TITLE
Timeout waiting for kernel_info reply

### ISSUE THREAD

**Issue by Eracro** [Calysto/octave_kernel#113]
Hey Everybody ! 
After a lot of attempts my octave kernel is not working when I launch a notebook and I still have the error  : 

`Timeout waiting for kernel_info reply from 1c44ea96-6725-4201-8ca0-0bee5b55473f`

- My octave (version 3.8.0) is working fine and I use Python 3.6. I am on OSX 10.12.5 

I inserted `export OCTAVE_EXECUTABLE=/usr/local/octave/3.8.0/bin/octave-3.8.0 ` into my ~/.bash_profile

- There is what I get with `$ jupyter-kernelspec` list :
```

Available kernels:
  octave      /Library/Frameworks/Python.framework/Versions/3.6/share/jupyter/kernels/octave
  python3    /Library/Frameworks/Python.framework/Versions/3.6/share/jupyter/kernels/python3

```

- Following another response I already tried : 

```
pip3 install metakernel
pip3 install pymatbridge
```

Thank you very much for your help !

**Comment by blink1073** [comment]
Hi @Eracro, what does the following show when run in `python`?

```python
import os
print(os.environ['OCTAVE_EXECUTABLE'])
from octave_kernel.kernel import OctaveEngine
e = OctaveEngine()
e.eval('ones(3)')
```

**Comment by Eracro** [comment]
Thank you for your answer ! 

I get : 
`/usr/local/octave/3.8.0/bin/octave-3.8.0`

**Comment by blink1073** [comment]
Was there no output from `e.eval('ones(3)')`?

**Comment by Eracro** [comment]
Actually `e = OctaveEngine()` seems to run for a long time I don't know if it's normal

**Comment by blink1073** [comment]
Can you share your `~/.octaverc` if you have one?

**Comment by Eracro** [comment]
`PS1('>> ')
cd '/Users/e/Library/Mobile Documents/com~apple~CloudDocs/Travail Etienne/ING4/Machine Learning'`

**Comment by blink1073** [comment]
mind moving that file to `~/.octaverc.bak` and trying the engine again?

**Comment by Eracro** [comment]
Yep ! 
Octave engine is running now : 

I have : 
```
e.eval('ones(3)') : 

'ans =\r\n\r\n   1   1   1\r\n   1   1   1\r\n   1   1   1\r\n\r\n'
```

**Comment by Eracro** [comment]
Octave Kernel is now running fine ! Thank you so much ! I would never have find out ! 
Thank you very much !

**Comment by blink1073** [comment]
Great!  I verified that `PS1` call was the problem.  Let's leave this issue open as a reminder to fix that if I can.

**Comment by blink1073** [comment]
Actually, looking at this again, `PS1` was not the problem, that was fixed in #108.
